djangosaml2/views.py

```diff
@@ -136,6 +136,13 @@ def login(request,
     selected_idp = request.GET.get('idp', None)
     conf = get_config(config_loader_path, request)
 
+    kwargs = {}
```

Comment:
Does kwargs need to get passed into a call somewhere within the login function? As of this commit, I cannot see anyplace where it is used.

Reply:
This way we can put in SAML_CONFIG['service']['sp']:
````
'force_authn': True
# and/or
'name_id_format_allow_create`: True
````

to have a SAML request with ForceAuthn and AllowCreate....

````
<ns0:AuthnRequest AssertionConsumerServiceURL="http://sp1.testunical.it:8000/saml2/acs/" Destination="http://idp1.testunical.it:9000/idp/sso/post" ForceAuthn="true" ID="id-J6EMSBhanmouUIjKr" IssueInstant="2019-05-23T20:38:32Z" ProtocolBinding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Version="2.0" xmlns:ns0="urn:oasis:names:tc:SAML:2.0:protocol" xmlns:ns1="urn:oasis:names:tc:SAML:2.0:assertion" xmlns:ns2="http://www.w3.org/2000/09/xmldsig#">

[...]

<ns0:NameIDPolicy AllowCreate="true" Format="urn:oasis:names:tc:SAML:2.0:nameid-format:persistent"/>
````

Those parameters are SAML standards. pySAML2 documentation here:
 - https://pysaml2.readthedocs.io/en/latest/howto/config.html#force-authn
 -  ... but nothing about allow_create ...

Reply:
Fixed, it was missed in the PR branch
see commits